SpatialGDK/Source/SpatialGDK/Private/Utils/EntityPool.cpp

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/EntityPool.h"
+#include "SpatialGDKSettings.h"
+#include "TimerManager.h"
+
+#include "Interop/SpatialReceiver.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialEntityPool);
+
+using namespace improbable;
+
+void UEntityPool::Init(USpatialNetDriver* InNetDriver, FTimerManager* InTimerManager)
+{
+	NetDriver = InNetDriver;
+	Receiver = InNetDriver->Receiver;
+	TimerManager = InTimerManager;
+
+	if (NetDriver->IsServer())
```

Comment:
Why is an EntityPool being created on the client?

Reply:
Fixed, NetDriver will now prevent creation of an EntityPool on a client